packages/fields/types/Content/Implementation.js

```diff
@@ -25,7 +25,9 @@ class Content extends Text {
       output: type,
     };
 
-    this.complexBlocks = this.config.blocks
+    this.blocks = Array.isArray(this.blocks) || [];
```

Comment:
Wouldn't `Array.isArray(this.blocks)` always evaluate to false since `this.blocks` wouldn't be defined at that point?

Reply:
Whoa, great catch! That whole line of code is all sorts of messed up! Not sure what I did there, but it's totally wrong 😓